ui/packages/security-idp-server/src/main/webapp/js/view/Login.view.js

```diff
@@ -88,8 +91,24 @@ define([
         }
       })
     },
-    showErrorText: function() {
-      this.$('#loginError').show()
+    showSpinnerAndDisableInput: function() {
+      $('#loginerrorup').hide()
+      $('#loginspinner').show()
+      $('#username').attr('readonly', true)
```

Comment:
I think you can use disabled instead of readonly, then do:

```javascript
var selectors = ['#username', '#password', '.btn-upsignin', '.btn-upclear', '.btn-pkisignin', '.btn-guestsignin']
selectors.forEach(function (selector) {
  $(selector).attr('disabled', true)
})
```
then you could use the same list for `hideSpinnerAndEnableInput`.